### PR TITLE
Update dg

### PR DESCRIPTION
**Pull request by MinkhantEain**

### CODE REVIEW

**Review by zhou-colla — APPROVED** [review]
Looks good to me!

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -310,7 +310,7 @@ Use case ends
 **MSS**
 1. User enters a command to edit all contacts with specific tags to new tag.
 2. Cher will change all contacts containing specified tags to the new tag.
-3. Cher will show the contacts that have been changed.<br>
+3. Cher will show the contacts that have the new tag.<br>
```
Comment (yyueyii) [inline docs/DeveloperGuide.md:313]: UC5 & UC7 are missing Cher save to disk

Comment (yyueyii) [reply docs/DeveloperGuide.md:313]: Also for UC5 and UC7 Extension, I think it's 2a. Cher detects error in input?

**Review by yyueyii — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (yyueyii) [reply docs/DeveloperGuide.md:313]: Also may be more specific to write Cher will show all contacts that have the new tag

Comment (MinkhantEain) [reply docs/DeveloperGuide.md:313]: I believe its 1a, Cher detects error in input as the error is caused by 1a

Comment (zhou-colla) [reply docs/DeveloperGuide.md]: 1a seems right since the validation is done right after the command input is entered

**Review by yyueyii — APPROVED** [review]
LGTM!

**Pull request merged**